sentry/src/db.rs

```diff
@@ -1,18 +1,28 @@
-use redis::aio::MultiplexedConnection;
-use redis::RedisError;
-
 use bb8::Pool;
 use bb8_postgres::tokio_postgres::NoTls;
 use bb8_postgres::PostgresConnectionManager;
+use redis::aio::MultiplexedConnection;
+use redis::RedisError;
+use std::env;
+
 use lazy_static::lazy_static;
 
 pub type DbPool = Pool<PostgresConnectionManager<NoTls>>;
 
 lazy_static! {
     static ref REDIS_URL: String =
-        std::env::var("REDIS_URL").unwrap_or_else(|_| String::from("redis://127.0.0.1:6379"));
-    static ref POSTGRES_URL: String = std::env::var("POSTGRES_URL")
-        .unwrap_or_else(|_| String::from("postgresql://postgres:postgres@localhost:5432"));
+        env::var("REDIS_URL").unwrap_or_else(|_| String::from("redis://127.0.0.1:6379"));
+    static ref POSTGRES_USER: String =
+        env::var("POSTGRES_USER").unwrap_or_else(|_| String::from("postgres"));
+    static ref POSTGRES_PASSWORD: String =
+        env::var("POSTGRES_PASSWORD").unwrap_or_else(|_| String::from("postgres"));
+    static ref POSTGRES_HOST: String =
+        env::var("POSTGRES_HOST").unwrap_or_else(|_| String::from("localhost"));
+    static ref POSTGRES_PORT: u16 = env::var("POSTGRES_PORT")
+        .unwrap_or("5432".to_string())
+        .parse()
+        .unwrap();
+    static ref POSTGRES_DB: Option<String> = env::var("POSTGRES_DB").ok();
```

Comment:
Wouldn't it be much better to have POSTGRES_URL as we did before and parse it for `migrant`. Having to pass several environment variables could become unfriendly quickly.

Reply:
Well then we need to copy-paste this in that case:
https://github.com/sfackler/rust-postgres/blob/master/tokio-postgres/src/config.rs#L667

Reply:
I don't want to do this, let's keep it simple and for now have separate env. variables. We can always improve later. `Migrant` is not helping as well as this is a result of it + tokio_postgres doesn't have the properties of the config publicly.